manifests/init.pp

```diff
@@ -81,24 +85,47 @@
     target => $log_path,
   }
 
-  wget::fetch { 'selenium-server-standalone':
-    source             => $jar_url,
-    destination        => "${jar_path}/${jar_name}",
-    timeout            => $download_timeout,
-    nocheckcertificate => $nocheckcertificate,
-    execuser           => $user,
-    require            => File[$jar_path],
+  if $package != undef {
+
+    package {$package:
+      ensure => installed,
+    }
```

Comment:
What platforms have a package for selenium?  There's not a lot of value here unless there is a default param name selected on a per platform basis.  What about adding a param to disable installation?  That would allow the package to be managed externally.